src/events/actions/updateWebsiteActions/generateVoxxrinJson.ts

```diff
@@ -0,0 +1,203 @@
+                primaryHex: string
+                primaryContrastHex: string
+                secondaryHex: string
+                secondaryContrastHex: string
+                tertiaryHex: string
+                tertiaryContrastHex: string
+            }
+        }
+        supportedTalkLanguages: { id: string; themeColor: string; label: string }[]
+        rooms: { id: string; title: string }[]
+        talkTracks: { id: string; title: string; themeColor: string }[]
+        talkFormats: { id: string; title: string; duration: string; themeColor: string }[]
+        features: {
+            favoritesEnabled: boolean
+            roomsDisplayed: boolean
+            remindMeOnceVideosAreAvailableEnabled: boolean
+            showInfosTab: true
+            showRoomCapacityIndicator: false
+            hideLanguages: []
+            ratings: {
+                scale: {
+                    enabled: boolean
+                }
+                bingo: {
+                    enabled: boolean
+                }
+                'free-text': {
+                    enabled: boolean
+                }
+            }
+        }
+        sponsors: {
+            type: string
+            typeBackgroundColor: string
+            typeFontColor: string
+            sponsorships: {
+                name: string
+                logoUrl: string
+                href: string
+            }[]
+        }[]
+        sessions: {
+            [key: string]: any
+        }
+        speakers: {
+            [key: string]: any
+        }
+    } = {
+        title: event.name,
+        headingTitle: event.name,
+        timezone: eventTimezone,
+        start: event.dates.start?.toISOString(),
+        end: event.dates.end?.toISOString(),
+        days: days.map((day) => ({
+            id: day.start.toISO() || Math.random().toString(),
+            localDate: day.start.toLocaleString(),
+        })),
+        logoUrl: logoUrl,
+        backgroundUrl: backgroundUrl,
+        theming: {
+            colors: {
+                primaryHex: event.color || '#F78125',
+                primaryContrastHex: hexContrast(event.color || '#F78125'),
+                secondaryHex: event.colorSecondary || '#3880FF',
+                secondaryContrastHex: hexContrast(event.colorSecondary || '#3880FF'),
+                tertiaryHex: event.colorBackground || '#202020',
+                tertiaryContrastHex: hexContrast(event.colorBackground || '#202020'),
+            },
+        },
+        supportedTalkLanguages: supportedTalkLanguages,
+        rooms: event.tracks.map((track) => ({ id: track.id, title: track.name })),
+        talkTracks: event.categories.map((category) => ({
+            id: category.id,
+            title: category.name,
+            themeColor: category.color || '#000000',
+        })),
+        talkFormats: event.formats.map((format) => ({
+            id: format.id,
+            title: format.name,
+            duration: `PT${format.durationMinutes}m`,
```

Comment:
@HugoGresse formats seems not to be exhaustive .. for instance, session `atelier-10-tests-simples-pour-` has a duration of `120min` and there is currently no format with a duration of `120min`

Reply:
this seems to also be an issue on the "human" part. I'll update the json so fix this.